test/specs/errors/conf/error_schema/error_schema.json

```diff
@@ -47,8 +47,114 @@
           "name": "Valid Table Name"
         }
       }
+    },
+    "main_delete_table": {
+      "comment": null,
+      "kind": "table",
+      "keys": [
+        {
+          "comment": null,
+          "annotations": {},
+          "unique_columns": [
+            "id"
+          ]
+        }
+      ],
+      "entityCount": 0,
+      "foreign_keys": [],
+      "table_name": "main_delete_table",
+      "schema_name": "error_schema",
+      "column_definitions": [
+        {
+          "comment": null,
+          "name": "id",
+          "default": null,
+          "nullok": false,
+          "type": {
+            "typename": "serial4"
+          },
+          "annotations": {}
+        },
+        {
+          "comment": null,
+          "name": "association_name",
+          "default": null,
+          "nullok": false,
+          "type": {
+            "typename": "text"
+          },
+          "annotations": {}
+        }
+      ],
+      "annotations": {
+        "tag:misd.isi.edu,2015:display": {
```

Comment:
Remove this annotation. Keep your schema as minimal as you can for your test so it can be easily readable.